### DISCUSSION TITLE
Upgrade instructions from 4.2.0-beta1 to 4.2.0-beta2

### DISCUSSION THREAD

**Discussion by solarisfire** [mastodon/mastodon#26597] [General]
Is it possible to get upgrade instructions for moving between the two versions?

I'd guess it's just:

git fetch && git checkout v4.2.0-beta2
bundle install and yarn install

I'd guess we don't need to do asset pre-compilation? Or do we?
I'd guess we only need to do the post deployment db migrations?

And then restart all the mastodon processes?
Anything with elasticsearch indexes?

**Comment by StrangeMinds1960** [comment]
I got into trouble doing just the git fetch, so I did the whole thing (including elasticsearch) and I'm back up & running again
